tests/product_selenium_unittest.py

```diff
@@ -0,0 +1,313 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import unittest
+import re
+import sys
+import os
+
+
+class ProductTest(unittest.TestCase):
```

Comment:
As commented in Functional Unit test for Product #1164, 'Add Finding' needs to be implemented before moving further. Everything else has been tested and verified. Good work.

Reply:
thanks..I hope to implement Add Finding as soon as possible.

Reply:
Add Finding completed